tests/functional/s3-versioning.sh

```diff
@@ -321,5 +324,36 @@ if ${AWS} s3api get-object --bucket "${BUCKET}" --key obj --version-id "$OBJ_2_E
     false
 fi
 
+echo "Check version contains only version"
+PUT_RES=$(${AWS} s3api put-object --bucket ${BUCKET} --key obj --body "${OBJ_2}")
+PUT_ID=$(jq -r ".VersionId|tonumber" <<< "$PUT_RES")
```

Comment:
Why create a new version? There is no check and this version is never deleted.

Reply:
Copy'n'Paste error